src/components/admin/settings-tab.js

```diff
@@ -142,12 +164,28 @@ function SettingsTab({ initialSettings, documents, onDirtyStateChange, onSetting
           onChange={handleLicenseChange}
           />
       </Card>
-      <Card className="SettingsTab-card SettingsTab-card--danger" title={t('createDocumentRegenerationBatchHeader')}>
+      <Card
+        className="SettingsTab-card SettingsTab-card--danger"
+        title={t('createDocumentRegenerationRequestHeader')}
+        extra={renderLastBatchExecution(lastDocumentRegenerationBatch)}
+        >
         <Button
-          onClick={handleCreateDocumentRegenerationBatchClick}
+          onClick={handleCreateDocumentRegenerationRequestClick}
```

Comment:
These names are such a mouthful 😅 
Would `handleStart[DocumentRegeneration/CdnResourceConsolidation]Click`, `startDocumentRegenerationHeader` (and so on) make sense too?

Reply:
True, changed them!

Reply:
Thanks! 😁 
We could probably streamline also `createDocumentRegenerationRequest[Header/Button]` and `cdnResourcesConsolidationRequest[Header/Button]`

Reply:
True (again), I changed them to `documentRegeneration[Header/Button]` and `cdnResourcesConsolidation[Header/Button]`, thanks!

Reply:
Poifect! 🤝